### PR TITLE
Expand demos to address #15 and some minor enhancements 

### PR DESCRIPTION
**Pull request by alshakero**
This PR is to address this comment https://github.com/Starcounter/underwear.css/issues/15#issuecomment-364402353. 

Each commit covers one bullet point from the comment. 

Easy view here: https://rawgit.com/Starcounter/underwear.css/Enhancements/index.html

### CODE REVIEW

**Comment by tomalec** [comment]
I wonder whether we should use any other styles than underwear. I think it would be nice to serve only underwear-ish experience, without light green color and dotted borders. If we need some styles, to for example position the top bar with logo, GH links etc. I think we may use inline styles, to make it extremely clean and obvious what comes from where.

Also, it would be good to have a list and the demo of custom properties available.

**Comment by alshakero** [comment]
> I wonder whether we should use any other styles than underwear. I think it would be nice to serve only underwear-ish experience, without light green color and dotted borders. If we need some styles, to for example position the top bar with logo, GH links etc. I think we may use inline styles, to make it extremely clean and obvious what comes from where.

I removed `style.css` and inlined very little CSS.

> Also, it would be good to have a list and the demo of custom properties available.

Could you elaborate on "custom properties"? I don't know what you mean.

**Comment by tomalec** [comment]
> Could you elaborate on "custom properties"? I don't know what you mean.

Something like this
https://github.com/Starcounter/uniform.css/pull/26/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR55

**Comment by tomalec** [comment]
Why do we need third input? I think it's redundant as does not show anything new.
```html
<input type="text" value="Default">
<input type="text" disabled value="Disabled">
<input type="text" value="Default" placeholder="&lt;place holder&gt;">
```

If we want to showcase a placeholder, I'd remove the value and `&lt;...&gt;`

File: index.html [hunk 1]
```diff
@@ -299,16 +453,124 @@ <h1>
...
+                    </label>
+                </div>
                 <div>
                     <button class="primary">Submit</button>
                     <button>Cancel</button>
                 </div>
             </div>
         </div>
+        <div>
+            <h1>Custom variables</h1>
```
Comment (tomalec) [inline index.html]: This is officially called  "Custom Properties", or less officially "CSS Variables" I'd stick to the official name in the header

Comment (warpech) [reply index.html]: Can we just stick to "CSS Custom Properties" which is the official name. According to the spec:

- this is a property: `--something`
- this is a variable: `var()`

File: index.html [hunk 2]
```diff
@@ -299,16 +453,124 @@ <h1>
...
         </div>
+        <div>
+            <h1>Custom variables</h1> 
+            <div>
+                <p>Underwear.css uses CSS Custom Properties (CSS Variables) that you can configure. You can also use them by yourself, to adhere to the consistent look and feel.</p>
+                <table>
+                    <tr><th>Name</th><th>Default value</th><th>Sample</th></tr>
+                    <tr>
+                        <td><code>--default-font-size</code></td>
+                        <td><code>14px</code></td>
```
Comment (warpech) [inline index.html]: I don't think it is sustainable to include the values as static text here. Perhaps you can populate it with a script?

Comment (alshakero) [reply index.html]: Done

**Comment by tomalec** [comment]
I'd rather keep demo a static file,  so one could see everything needed after pressing `ctrl+u`.

To make it easier for us to maintain custom props, we could pre-process index.html

File: index.html [hunk 3]
```diff
@@ -54,8 +54,7 @@
         <div>
             <div>
                 <h1>
-                    <span class="emoji">👙</span> Underwear
-                    <span class="suffix">.css</span>
+                    <span class="emoji">👙</span> Underwear<span class="suffix">.css</span>
```
Comment (tomalec) [inline index.html]: could you remove those classes, they are not defined anywhere

**Comment by alshakero** [comment]
Now `index.html` is automatically generated.

**Comment by alshakero** [comment]
Awesome. I won't tag or release I'll fix many issues today.

**Pull request merged**